api/api_public_klay.go

```diff
@@ -70,3 +75,61 @@ func (s *PublicKlayAPI) Syncing() (interface{}, error) {
 		"knownStates":   hexutil.Uint64(progress.KnownStates),
 	}, nil
 }
+
+// EncodeAccountKey gets an account key of JSON format and returns RLP encoded bytes of the key.
+func (s *PublicKlayAPI) EncodeAccountKey(accKey accountkey.AccountKeyJSON) (hexutil.Bytes, error) {
+	buffer := new(bytes.Buffer)
+	if accKey.KeyType == nil {
+		return nil, errors.New("key type is not specified")
```

Comment:
```suggestion
		return nil, errNoKeyType
```

Reply:
It could be, but I wanna give a different error message to users. 
Since the JSON input can have a complicated format, users may need detailed error messages to prevent mistakes.